docs/articles/baseplatform/reproducing-yocto-builds.rst

```diff
@@ -0,0 +1,91 @@
+Reproducing yocto builds
+========================
+
+This document describes how to:
+
+* Configure a ``Yocto`` build so that it can be reproduced.
+* Reproduce such a ``Yocto`` build. The reproduction requires that additional artifacts are archived for the Yocto build that is to be reproduced.
+
+Intro to Yocto build reproducibility
+------------------------------------
+
+A bitbake recipe specifies where the source code of a package should be downloaded from. Because there are several ways to do this, a recipe might not specify a specific version but rather the head of some branch or a "latest"-release. Even though this is not recommended, there is no guarantee that developers will not write recipes this way. This of course affects reproducibility as the maintainer of some upstream project may update the head of the branch or the "latest"-release between builds so that the same version of a recipe yields different results. In addition, in cases when reproducibility of a Yocto build is required, the actual availability of the upstream source code packages might cause a problem, e.g. if a tar-archived is moved to some other URL.
```

Comment:
"...the same version of a recipe..." -> "...the same recipe compiled at different times..."

"tar-archived" -> "tar archive" (unless you meant something else?)